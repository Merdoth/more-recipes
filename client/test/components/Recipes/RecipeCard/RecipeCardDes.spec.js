import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { RecipeCardDes } from
  '../../../../components/Recipes/RecipeCard/RecipeCardDes.jsx';

const setup = () => shallow(<RecipeCardDes/>);

describe('RecipeCardDes Component snapshot', () => {
  it('it should render the right amount of elements', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
