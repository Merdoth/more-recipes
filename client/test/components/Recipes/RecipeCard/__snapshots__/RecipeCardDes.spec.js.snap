// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeCardDes Component snapshot it should render the right amount of elements 1`] = `
<div
  className="des text-truncate"
>
  <h2 />
  <h4 />
</div>
`;
